Header header 

#LaserScan:
float32[] distances
float32[] angles

#Camera:
uint8[] pixel_values